src/test/groovy/edu/ie3/simona/model/participant/PvModelTest.groovy

```diff
@@ -473,9 +473,9 @@ class PvModelTest extends Specification {
     // 0.244 MJ/m^2 = 67.777778 Wh/m^2
     //Beam Radiation on horizontal surface
     Irradiation eBeamH = Sq.create(67.777778d, WattHoursPerSquareMeter$.MODULE$)
-    // 0.769 MJ/m^2 = 213,61111 Wh/m^2
+    // 0.796 MJ/m^2 = 221,111288 Wh/m^2
```

Comment:
This is a typo we made at some point, so this correction should find its way into the new `PvModelSpec` as well.